google-cloud-spanner/lib/google/spanner/v1/spanner_pb.rb

```diff
@@ -10,7 +10,6 @@
 require 'google/protobuf/timestamp_pb'
 require 'google/spanner/v1/keys_pb'
 require 'google/spanner/v1/mutation_pb'
-require 'google/spanner/v1/query_plan_pb'
```

Comment:
This was directly from the proto compiler. I'll do some digging when I get back from lunch.